test/navigation/ItemsNavigatorSpec.scala

```diff
@@ -347,28 +332,13 @@ class ItemsNavigatorSpec extends SpecBase {
 
     "in Check mode" - {
       "must go from the Excise Product Code page" - {
-
-        //TODO: Route to CAM-ITM43 when implemented
-        "to CAM-ITM43 page" - {
-          Seq("S500", "T300", "S400", "E600", "E800", "E910").foreach(epc => {
-            s"when the EPC is $epc" in {
-              navigator.nextPage(
-                ItemExciseProductCodePage(testIndex1),
-                CheckMode,
-                emptyUserAnswers.set(ItemExciseProductCodePage(testIndex1), epc)) mustBe
-                testOnly.controllers.routes.UnderConstructionController.onPageLoad()
-            }
-          })
-        }
-
```

Comment:
as above

Reply:
Related to my comment above - these EPCs shouldn't be treated differently as far as the navigator is concerned.